packages/hash/subgraph/jest.config.cjs

```diff
@@ -0,0 +1,23 @@
+/** @type {import('jest').Config} */
+module.exports = {
+  collectCoverage: process.env.TEST_COVERAGE === "true",
+  collectCoverageFrom: [
+    "**/*.{c,m,}{j,t}s{x,}",
+    "!**/node_modules/**",
+    "!**/dist/**",
+  ],
+  coverageReporters: ["lcov", "text"],
+  preset: "ts-jest",
+  testEnvironment: "node",
+  // recreating DB takes longer than the default 5 seconds.
+  // The chosen default give a lot of room to the integration test.
+  testTimeout: 60000,
```

Comment:
We will be using DB in the tests in a follow-up PR so I thought I'd just leave this in for consistency with the other tests to avoid accidentally forgetting to re-add it. However, it's likely we won't be reseeding the DB each test, so this is probably redundant.

Have removed, will re-add if necessary, thanks!

Reply:
> We will be using DB in the tests in a follow-up PR

Oh I wasn't aware of this, why would this package which is concerned with the `Subgraph` typing and mapping functionality need database access?

Reply:
> We'll actually need data to test those.

Sure, I just wasn't aware that would require database access as we could just declare the test data statically, to help decouple the subgraph library from the hash app. Don't feel strongly about this though, just explaining why I didn't forsee a database connection being required here.

Reply:
Yes, I wasn't blocking approval on resolution of this thread just wanted you to see my reply